### ISSUE TITLE
Inherited gdscript fails to call _process or _physics_process from parent

### ISSUE THREAD

**Issue by pouleyKetchoupp** [godotengine/godot#44437]
**Godot version:**
Master 92d4deedaa542e9458845fdf2b2a44fedd19fde8

**OS/device including version:**
Windows 10

**Issue description:**
When an inherited script uses `super(delta)` in `_process(delta)` to call the parent script's `_process(delta)` the debugger breaks with the error `compiler bug, function name not found`.

**Steps to reproduce:**
1. Create script A which implements `_process`
2. Create script B which inherits from script A
3. Make script B implement `_process` and call `super(delta)`
4. Run
5. Observe debugger break

**Minimal reproduction project:**
[bug-super-call-4.0.zip](https://github.com/godotengine/godot/files/5704375/bug-super-call-4.0.zip)

**Comment by AndreaCatania** [comment]
I can reproduce it too.

**Comment by lyuma** [comment]
reproduction project seems fixed on latest master 85186bc:

Can someone confirm if this is the correct output?
```
Godot Engine v4.0.dev.custom_build.85186bc0c - https://godotengine.org
Vulkan API 1.2.162
WARNING: GENERAL - Message Id Number: 0 | Message Id Name: Loader Message
        ReadDataFilesInRegistry: Registry lookup failed to get layer manifest files.
        Objects - 1
                Object[0] - VK_OBJECT_TYPE_INSTANCE, Handle 2160701320432
     at: VulkanContext::_debug_messenger_callback (drivers\vulkan\vulkan_context.cpp:154)
Using Vulkan Device #0: NVIDIA - NVIDIA GeForce RTX 2080 Ti

ParentNode: _ready()
Node: _ready()
ParentNode: _process()
Node: _process()
ParentNode: _process()
Node: _process()
ParentNode: _process()
Node: _process()
ParentNode: _process()
Node: _process()
```
...

**Comment by pouleyKetchoupp** [comment]
Yes I can confirm it's fixed on 0cee8831b25377567bff8f46f7eaad8cb203dbf8, closing.
